schemas/collaborations.yml

```diff
@@ -21,11 +21,12 @@ id: "enum(
           'serre-lab',
           'creton-lab',
           'crunch-group',
-	  'kantor-lab',
-	  'mcdonald-lab',
-	  'mchugh-lab',
-	  'rothman-lab',
-	  'spade-lab'
+          'kantor-lab',
+          'mcdonald-lab',
+          'mchugh-lab',
+          'rothman-lab',
+          'spade-lab',
+          'musslick'
```

Comment:
I'd say if the title of the collaboration is AER, then `aer-group` is more consistent here

Reply:
Also done